supervisor/supervisor.go

```diff
@@ -48,9 +48,6 @@ func New(stateDir string, runtimeName, shimName string, runtimeArgs []string, ti
 	}
 	go s.exitHandler()
 	go s.oomHandler()
-	if err := s.restore(); err != nil {
```

Comment:
sorry, it's a mistake, I forget that restore need be called before the start of grpc service.